### PR TITLE
⚠️ Add pre_deploy command to check imports/exports

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/utils/management/commands/pre_deploy.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+from temba.flows.models import ExportFlowResultsTask
+from temba.msgs.models import ExportMessagesTask
+
+
+class Command(BaseCommand):
+    help = "Runs pre-deployment checks"
+
+    def handle(self, *args, **kwargs):
+        unfinished_tasks = {
+            "contact-import": ImportTask.objects.filter(task_status__in=(ImportTask.STARTED, ImportTask.RUNNING)),
```
Comment (rowanseymour) [inline temba/utils/management/commands/pre_deploy.py]: intentionally not filtering by date here.. am thinking we should start considering imports/exports left in a "running" state as something to fix..

Comment (rowanseymour) [reply temba/utils/management/commands/pre_deploy.py]: I'm feeling paranoid about tasks getting into a state where they repeatedly error and get retried, and there's no visibility of that so it continues forever.

**Comment by nicpottier** [comment]
Isn't this going to give weird results across boxes? IE, you ship to `text1`, it is going to report things that are actually still running at that point in time on `text2`. Same when you ship to `text2`.

Just seems like looking only at status without any concept of how long it has been in that status isn't quite enough data.

**Comment by rowanseymour** [comment]
As I said in https://github.com/nyaruka/rapidpro/pull/2708#discussion_r385231615, I think we need to start treating tasks being left as unfinished as a problem to fix - need to make sure all error cases result in the task being marked as failed. The output includes how long ago the task was started so we can still use that to guage whether the task is actually still running.

Regarding different boxes, we could lookup the host that each running task in supposedly running. I was thinking of this as imperfect first step, toward not deploying to any box when there are running tasks.

**Comment by nicpottier** [comment]
I guess maybe it detecting tasks across boxes could be considered a feature in that we really probably don't want any significant period of having deployed to only one box. So sure, I guess this is a step towards better understanding the impact of deploying.

**Pull request merged**